authorizer/scraper.go

```diff
@@ -11,13 +11,15 @@ var _ influxdb.ScraperTargetStoreService = (*ScraperTargetStoreService)(nil)
 // ScraperTargetStoreService wraps a influxdb.ScraperTargetStoreService and authorizes actions
 // against it appropriately.
 type ScraperTargetStoreService struct {
+	influxdb.UserResourceMappingService
```

Comment:
I'm a bit uneasy about this being embedded like this. What's the reasoning for doing this?

Reply:
I would like to have the urm as part of the same transaction when creating the scraper. the user should become the owner of the scraper once created. also when delete the scraper, all the related urm should be removed as well

Reply:
I did the same thing of telegraf as well.

Reply:
Why embed it though?